client/state/login/selectors.js

```diff
@@ -27,9 +20,8 @@ export const getTwoFactorUserId = ( state ) => {
  * @param	{string}	nonceType nonce's type
  * @returns {?string}         The nonce.
  */
-export const getTwoFactorAuthNonce = ( state, nonceType ) => {
-	return state.login.twoFactorAuth[ `two_step_nonce_${ nonceType }` ] ?? null;
-};
+export const getTwoFactorAuthNonce = ( state, nonceType ) =>
+	state.login.twoFactorAuth[ `two_step_nonce_${ nonceType }` ] ?? null;
```

Comment:
Looks like there's a pre-existing bug here: `twoFactorAuth` can be `null`, so the property access should be optional: ``twoFactorAuth?.[ `two_step_...` ]``

Reply:
Good catch, done!